catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogOperations.java

```diff
@@ -73,18 +77,27 @@
 import org.apache.hadoop.fs.FileStatus;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
-import org.awaitility.Awaitility;
-import org.awaitility.core.ConditionTimeoutException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class HadoopCatalogOperations extends ManagedSchemaOperations
     implements CatalogOperations, FilesetCatalog {
   private static final String SCHEMA_DOES_NOT_EXIST_MSG = "Schema %s does not exist";
   private static final String FILESET_DOES_NOT_EXIST_MSG = "Fileset %s does not exist";
+  private static final ThreadPoolExecutor FILE_SYSTEM_EXECUTOR =
+      new ThreadPoolExecutor(
+          2,
+          Math.max(2, Runtime.getRuntime().availableProcessors() / 2),
+          60L,
+          TimeUnit.SECONDS,
+          new LinkedBlockingQueue<>(100),
```

Comment:
Size 100 is very small, assuming there are a lot of requests sent to the Gravitino server, as the core thread size is `Math.max(2, Runtime.getRuntime().availableProcessors() / 2)`, as long as the concurrency exceeds that value, the request will be put the queue and queue will be easy full.